Datahub.Web/.env.example

```diff
@@ -1,6 +1,16 @@
-﻿ES_ENDPOINT_HOST=search-jncc-website-live-search-abcdefghijklm.eu-west-1.es.amazonaws.com
+﻿
+# connect to local dockerized elasticsearch
+ES_ENDPOINT_HOST=localhost
+ES_ENDPOINT_SCHEME=http
+ES_ENDPOINT_PORT=9200
+
+# or, connect to aws elasticsearch
+ES_ENDPOINT_HOST=search-jncc-website-live-search-abcdefghijklm.eu-west-1.es.amazonaws.com
```

Comment:
I wouldn't put this form of URL in even if fake, makes it easier to miss someone accidentally committing the real URL unnoticed in the future.

Reply:
I think I disagree, but see what you mean. The benefit of having a syntactically correct example when you're trying to create a `.env` file from the `.env.example` file is quite significant.

Reply:
Here's a good compromise - the best of both worlds?4b76fcb71297f73befce2dff3cb35402b27822d9
Thanks, appreciate the call out.  😌